src/Annotation/README.md

```diff
@@ -54,6 +54,43 @@ Display informative text related to an object on screen. Unlike the tooltip an a
 </>
 ```
 
+### Basic Usage (RTL)
```

Comment:
Curious about the need to change anything from the props API / CSS utility classes as a consumer to support RTL on this component?

Reply:
added to `scss/core/_utilities.scss` new rtl rules